src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/ScopeKeys.cs

```diff
@@ -3,19 +3,19 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Loggers
```

Comment:
Seems like the namespace should be Microsoft.Azure.WebJobs.Logging. That's the default namespace for the logging assembly